### PR TITLE
[eclipse/xtext-core#1499] Stricter JavaDoc validation on org.eclipse.xtext.ui

### PR DESCRIPTION
**Pull request by szarnekow**
see eclipse/xtext-core#1499

### CODE REVIEW

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/model/ResourceForIEditorInputFactory.java [hunk 1]
```diff
@@ -133,7 +133,6 @@ protected boolean isValidationDisabled(java.net.URI uri) {
 	}
 
 	/**
-	 * @param uri
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/model/ResourceForIEditorInputFactory.java:136]: ?

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/XtextUIMessages.java [hunk 2]
```diff
@@ -39,7 +39,6 @@ private XtextUIMessages() {
 	}
 
 	/**
-	 * @param key
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/XtextUIMessages.java:42]: so missing params are not complained?

Comment (szarnekow) [reply org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/XtextUIMessages.java:42]: So far only *present* comments are validated. Enabling this for missing params would be ... quite some effort to be fixed.

I'd like to approach this step-by-step.
Unfortunately there is no such thing as validating only the touched files.

File: org.eclipse.xtext.ui/.settings/org.eclipse.jdt.core.prefs [hunk 3]
```diff
@@ -67,7 +67,7 @@ org.eclipse.jdt.core.compiler.problem.missingHashCodeMethod=warning
 org.eclipse.jdt.core.compiler.problem.missingJavadocComments=ignore
 org.eclipse.jdt.core.compiler.problem.missingJavadocCommentsOverriding=disabled
 org.eclipse.jdt.core.compiler.problem.missingJavadocCommentsVisibility=public
-org.eclipse.jdt.core.compiler.problem.missingJavadocTagDescription=return_tag
+org.eclipse.jdt.core.compiler.problem.missingJavadocTagDescription=all_standard_tags
 org.eclipse.jdt.core.compiler.problem.missingJavadocTags=ignore
```
Comment (szarnekow) [inline org.eclipse.xtext.ui/.settings/org.eclipse.jdt.core.prefs:71]: @cdietrich here is the setting missingJavadocTags=ignore but if present, validate them

**Pull request merged**